sale_order_invoicing_queued/models/sale_order.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+try:
+    from odoo.addons.queue_job.job import job
+except ImportError:
```

Comment:
You can add a TODO note to remove this try-except in v12.

Reply:
Same answer.

Reply:
Same as what? :confused: